content/community/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Community"
+weight: 100
+---
+
+# Contributing to Paketo Buildpacks
+
+## How can I contribute to the project?
```

Comment:
```suggestion
```
This feels redundant since the page is titled "Contributing to Paketo Buildpacks"